docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -669,20 +713,23 @@ When a creature moves to a position that makes it more difficult to hit them, th
 gain the effects of cover.
 
 **Partially Covered**: When a creature is still visible behind something but it is still
-  reasonable to hit them. While partially covered the creature gains +1 AR. For example...
+  reasonable to hit them. While partially covered the creature's AR is reduced by -1,
+  making them harder to hit. For example...
 
 - Standing directly between another creature and an attacker.
 - Standing behind a crate or barrel that is half the creature's height.
 
 **Fully Covered**: when a creature is nearly impossible to see behind something and it
-  is unlikely to land a hit on them. While fully covered, the creature gains +2 AR. For example...
+  is unlikely to land a hit on them. While fully covered, the creature's AR is reduced
```

Comment:
What's the difference between character and creature? If relevant, please specify. It not, please revise for consistency

Reply:
We currently have 110 references to `character` throughout the PGF and 41 references to `creature`.

`creature` is mostly referenced when talking about mechanics like Status Conditions, Cover, Lighting, and Combat Rules. Character is used everywhere else. 

Should everything refer to `character` instead of `creature`? or do we indicate the difference in some way where a `character` is a humanoid with the ability to communicate whereas a `creature` is anything present in the game world? Not sure how we'd clarify the difference, if any, but open to feedback.

Reply:
Sounds like the treading the same ground as `Dealer`, `Bystander`, `NPC`. You could pick `Character` as a hypernym and add it to the Glossary. Looks like NPC might need adding there too